securedrop_client/message_sync.py

```diff
@@ -70,7 +70,8 @@ def __init__(self, api, home, is_qubes):
 
     def run(self, loop=True):
         while True:
-            submissions = storage.find_new_submissions(self.session)
+            submissions = storage.find_new_messages(self.session)
+            submissions.extend(storage.find_new_files(self.session))
```

Comment:
so I missed this in review: we don't want the message fetching to extend to files (these can be huge). I'm going to remove in #262

Reply:
Ah right. Sorry. I forgot to update this when we changed that filter on the `find_new_submissions` function.